src/main/java/model/Transaction.java

```diff
@@ -0,0 +1,42 @@
+package model;
+
+import service.SeparatorService;
+import service.impl.SeparatorServiceImpl;
+import strategy.OperationHandler;
+
+public class Transaction {
+    private int value;
+    private String fruit;
+    private OperationHandler operationHandler;

```

Comment:
Handler can’t  be characteristic of the object 
```suggestion
    private Operation operation;

```